src/protocol/taiko_alethia/TaikoInbox.sol

```diff
@@ -26,34 +28,44 @@ contract TaikoInbox {
         maxAnchorBlockIdOffset = _maxAnchorBlockIdOffset;
     }
 
-    function publish(uint256 nBlobs, bytes calldata data, uint64 anchorBlockId) external {
+    function publish(uint256 nBlobs, uint64 anchorBlockId) external {
         if (address(lookahead) != address(0)) {
             require(lookahead.isCurrentPreconfer(msg.sender), "not current preconfer");
         }
```

Comment:
Isn't this redundant? The lookahead should return false for the `address(0)`

Reply:
I think that's true for a low-level call. The solc compiler will add a check to ensure `lookahead` is a valid contract and revert if not.